### PR TITLE
fixes humphd/brackets#124 - replace NoHostServer for HTML and Static Servers

### PR DESCRIPTION
**Pull request by K-LV**
WIP. Patch for [humphd/brackets#124](https://github.com/humphd/brackets/issues/124). Replaces NoHostServer for HTMLServer and NewStaticServer.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -25,33 +25,49 @@ define(function (require, exports, module) {
...
+        _staticServer,
         codeMirror,
-        fs           = appshell.Filer.fs(),
         parentWindow = window.parent,
         params       = new UrlParams();
 
     // Load initial document
     var defaultHTML = require("text!lib/default.html");
 
+    var file = FileSystem.getFileForPath("/index.html");
```
Comment (humphd) [inline main.js]: Move this into `appReady` just before you need it vs. out here.

File: main.js [hunk 2]
```diff
@@ -95,7 +111,7 @@ define(function (require, exports, module) {
             // XXXhumph - this depends on setLauncher() from https://github.com/adobe/brackets/pull/10558
```
Comment (humphd) [inline main.js]: Remove this comment, it's landed upstream now.

File: main.js [hunk 3]
```diff
@@ -158,9 +175,14 @@ define(function (require, exports, module) {
             type: "bramble:loaded"
         }), "*");
 
-        //Below are methods to change the preferences of breackets, more available at:
+        // Below are methods to change the preferences of brackets, more available at:
         // https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#list-of-supported-preferences
         PreferencesManager.set("insertHintOnTab", true);
+        // Make the spaceUnits and tabSize consistent
```
Comment (humphd) [inline main.js:178]: What are these changes doing in here?  Do you need to rebase?

**Comment by humphd** [comment]
A few things to do here, show me again when it's done and I'll re-read.

**Comment by humphd** [comment]
r+

**Comment by K-LV** [comment]
@humphd updated main.js to use StaticServer vs NewStaticServer

File: main.js [hunk 4]
```diff
@@ -25,15 +25,16 @@ define(function (require, exports, module) {
         // Load nohost dependencies
         Browser              = require("lib/iframe-browser"),
         HideUI               = require("lib/hideUI"),
-        Launcher             = require("lib/launcher").Launcher,
-        NoHostServer         = require("nohost/src/NoHostServer").NoHostServer,
+        Launcher             = require("lib/launcher"),
+        HTMLServer           = require("nohost/src/HTMLServer").HTMLServer,
+        StaticServer      = require("nohost/src/StaticServer").StaticServer,
```
Comment (humphd) [inline main.js]: Fix indent here to align `=` sign.

**Comment by humphd** [comment]
One nit, r+ with that.

**Pull request merged**